### PR TITLE
Handle unknown compilers nicer, fallback to passthrough

### PR DESCRIPTION
**Pull request by MarshallOfSound**
Fixes electron-userland/electron-forge/issues/53

### CODE REVIEW

File: src/compiler-host.js [hunk 1]
```diff
@@ -239,7 +239,11 @@ export default class CompilerHost {
     if (!compiler) {
       compiler = this.fallbackCompiler;
 
-      let { code, binaryData, mimeType } = await compiler.get(filePath);
+      if (!compiler) {
+        compiler = this.getPassthroughCompiler();
```
Comment (paulcbetts) [inline src/compiler-host.js]: I'm not sure that this is correct, for complicated weird reasons. Do you have a repro file that you hit this with?

Comment (MarshallOfSound) [inline src/compiler-host.js]: @paulcbetts I used this rep for repro

https://github.com/manikantag/electron-compile-css-issue

Comment (MarshallOfSound) [inline src/compiler-host.js]: Just running `electron-forge package` and then launching the packaged app

Comment (paulcbetts) [inline src/compiler-host.js]: Cool. Imma fix this today

**Pull request closed without merging**